src/styles/utils/_extends.scss

```diff
@@ -0,0 +1,14 @@
+// code here
+@import "./mixins";
+@import "./variables";
+
+%watch-center {
+  @include center($center-size);
+  background-color: $c-white;
+}
+
+%clock-face-centering {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+}
```

Comment:
No need to import mixins and variables here, just reorder your imports in `main.scss`.

You're using these placeholders only once. Maybe they're unnecessary?

Reply:
yup. i used only once, but maby in the future i'll use it again